src/style.css

```
@@ -1 +1,63 @@
-/* Styles go here */
+html {
```

```suggestion
* {
```
`html` selector affects only for `html` tag

---

I don't understand because in tasks we don't use *